swift/ql/test/extractor-tests/generated/decl/ConcreteVarDecl/ConcreteVarDecl_getParentPattern.expected

```diff
@@ -1,3 +1,4 @@
+| var_decls.swift:2:7:2:7 | i | var_decls.swift:2:7:2:7 | i |
```

Comment:
it's actually weird that this was missing! This is the `for` loop `i` variable, which is distinct from the `i` variable introduced after the loop.

Reply:
ah, wait, this is a parent pattern thing, I don't fully know what to make of this :laughing: